ecommerce_api_client/tests/test_auth.py

```diff
@@ -1,38 +1,67 @@
 # -*- coding: UTF-8 -*-
+
+import datetime
+import requests
 from unittest import TestCase
 
 import httpretty
 import jwt
-import requests
```

Comment:
@clintonb Sorry.. I sort of assume seeing it wasn't in requirements.txt. Will revert.